impl/src/main/java/org/ehcache/impl/config/loaderwriter/DefaultCacheLoaderWriterProviderConfiguration.java

```diff
@@ -24,7 +24,15 @@
 /**
  * {@link ServiceCreationConfiguration} for the default {@link CacheLoaderWriterProvider}.
  */
-public class DefaultCacheLoaderWriterProviderConfiguration extends ClassInstanceProviderConfiguration<String, DefaultCacheLoaderWriterConfiguration> implements ServiceCreationConfiguration<CacheLoaderWriterProvider> {
+public class DefaultCacheLoaderWriterProviderConfiguration extends ClassInstanceProviderConfiguration<String, DefaultCacheLoaderWriterConfiguration> implements ServiceCreationConfiguration<CacheLoaderWriterProvider, DefaultCacheLoaderWriterProviderConfiguration> {
+
+  public DefaultCacheLoaderWriterProviderConfiguration() {
```

Comment:
This constructor is used only in tests -- do we really need it?

Reply:
It's the implicit no-arg constructor that always used to exist... so yes?